frontend/src/components/grouping/FuzzyMatchNotificationGroupingComponent.vue

```diff
@@ -229,6 +234,7 @@ const clearNotification = () => {
   fuzzyMatchedError.value.show = false;
   fuzzyMatchedError.value.fuzzy = false;
   fuzzyMatchedError.value.matches = [];
+  customError.value = {};
 };
 
 const handleFuzzyErrorMessage = (event: FuzzyMatcherEvent | undefined, _payload?: any) => {
```

Comment:
The variable `_payload` had that leading `_` just to signal for linters that we really intended not to use it.
At the same time I left it there because I imagined it could be helpful at some time.
Anyway... Can you please remove that character now that we are using the variable?